app/views/pages/trade_remedies.html.erb

```diff
@@ -1,13 +1,9 @@
 <%= link_to('Back', country_of_origin_path, class: "govuk-back-link") %>
 
 <main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
```

Comment:
This also needs to go I think. Have a look at the layouts/application.html.

Reply:
You're right, it's duplicated.

I've also added a spacer around the "back" link to make it look less cramped (the 'main' was adding some padding)

No spacer:

<img width="789" alt="Screen Shot 2021-05-24 at 10 45 53" src="https://user-images.githubusercontent.com/2742327/119330286-f6a6ea00-bc7d-11eb-80e7-b7e7fad58e62.png">

With spacer:

<img width="791" alt="Screen Shot 2021-05-24 at 10 45 31" src="https://user-images.githubusercontent.com/2742327/119330318-fdcdf800-bc7d-11eb-8449-1d5db0821dbf.png">